src/test/java/com/hocs/test/pages/give_me_a_case/fetch.java

```diff
@@ -43,6 +50,69 @@
 
     Workstacks workstacks;
 
+    Page page;
```

Comment:
Extends page so don't need this declaration, just use the features as if you're in the page class already.